src/BizHawk.Emulation.Cores/Consoles/Nintendo/N64/NativeApi/mupen64plusInputApi.cs

```diff
@@ -70,14 +70,12 @@ public mupen64plusInputApi(mupen64plusApi core)
 			InpDll = core.AttachPlugin(mupen64plusApi.m64p_plugin_type.M64PLUGIN_INPUT,
 				"mupen64plus-input-bkm.dll");
 
-			mupen64plusApi.m64p_error result;
 			InpSetInputCallback = GetInputDelegate<SetInputCallback>("SetInputCallback");
 			InpSetRumbleCallback = GetInputDelegate<SetRumbleCallback>("SetRumbleCallback");
 			InpSetControllerPakType = GetInputDelegate<SetControllerPakType>("SetControllerPakType");
 			InpSetControllerConnected = GetInputDelegate<SetControllerConnected>("SetControllerConnected");
 
-			m64pRumbleCallback = new RumbleCallback(FireOnRumbleChange);
-			result = InpSetRumbleCallback(m64pRumbleCallback);
+			SetM64RumbleCallback(new RumbleCallback(FireOnRumbleChange));

```

Comment:
I didn't do this (712dbc041) in the old branch, what's its purpose?

Reply:
I think it's cleaner this way.